ci/scenarios/vPool/add_remove_vpool_concurrently/main.py

```diff
@@ -73,36 +71,39 @@ def add_vpools(self):
                                                    'storage_ip': sr.ip,
                                                    'storagerouter_ip': sr.ip,
                                                    'vpool_name': vpool_name,
-                                                   'writecache_size': 10}})
+                                                   'writecache_size': 5}})
             tasks[vpool_name] = self.api.post(api='storagerouters/{0}/add_vpool'.format(sr.guid), data=data)
-
         for vpool_name, task in tasks.iteritems():
```

Comment:
Please use more meaningful names. `task` is an ID, so `task_id` would be more appropriate here